server.js

```diff
@@ -1,7 +1,26 @@
 const app = require("./app");
+const mongoose = require("mongoose");
 
 const PORT = process.env.PORT || 3000;
 
 app.listen(PORT, () => {
   console.log("Server running. Use our API on port: 3000");
 });
+
+const MONGODB_URI =
+  "mongodb+srv://97olka:<password>@cluster0.q4ue9io.mongodb.net/";
```

Comment:
To powinno być w .env

Reply:
Zrób jeszcze .env.example który będzie przykładową listą zmiennych:
PORT=
MONGODB_URI=